src/PolyhedralGeometry/Polyhedron/standard_constructions.jl

```diff
@@ -866,7 +900,39 @@ julia> n_facets(T)
 12
 ```
 """
-catalan_solid(s::String) = polyhedron(Polymake.polytope.catalan_solid(s))
+function catalan_solid(s::String)
+  if s in _catalan_strings_from_oscar
+    # code used for generation of loaded files can be found at:
+    # https://github.com/dmg-lab/RegularSolidsSrc
+    return getfield(Main, Symbol(s))()
+  end
+  pmp = Polymake.polytope.catalan_solid(s)
+  return polyhedron(pmp)
+end
+
+@doc raw"""
+      pentagonal_icositetrahedron()
+
+Construct the Pentagonal Icositetrahedron, a Catalan solid.
```

Comment:
```suggestion
Construct the pentagonal icositetrahedron, a Catalan solid.
```
Similar changes need to be made in multiple other places: Catalan, Johnson, Archimedes are names and should stay capitalized, but generic nouns and adjectives like `snub` or `Hexecontahedron` etc. should be lowercase.